data/distro.yml

```diff
@@ -794,3 +794,33 @@
         <p>Note: graphical installation of Flatpak apps may not be possible with GNU Guix.</p>
       </li>
     </ol>
+    
+    
+- name: "PCLinuxOS"
+  logo: "pclinuxos.svg"
+  info: >
+    <ol class="distrotut">
+      <li>
+        <h2>Install Flatpak</h2>
+        <p>The flatpak package is available in the PCLinuxOS repositories, it can be installed using Synaptic Package Manager or by running the following in the terminal:</p>
```

Comment:
```suggestion
        <p>The flatpak package is available in the PCLinuxOS repositories, it can be installed using Synaptic Package Manager or by running the following in a terminal:</p>
```